src/components/dashboard.js

```diff
@@ -75,7 +75,7 @@ export default class DashBoard extends Component {
     }
     // implement pagination
     onPrevious() {
-        axios.get('http://127.0.0.1:5000' + this.state.previousPage, {
+        axios.get('https://shoppinglist-apis.herokuapp.com' + this.state.previousPage, {
             headers: {
                 'Content-type': 'application/x-www-form-urlencoded',
                 'Authorization': localStorage.getItem("token")
```

Comment:
It is not advisable to get `access_token` inside axios methods, I would recommend you do this outside axios method.